packages/fxa-content-server/server/config/local.json-dist

```diff
@@ -73,6 +73,6 @@
   },
   "rolloutRates": {
     "keyStretchV2": 1,
-    "generalizedReactApp": 1
+    "generalizedReactApp": 0
```

Comment:
Why does this need to be 0?

Reply:
Eh, we could leave it as 1 but this PR removes email-first from the experiment (so we have no active react experiment until we work on the next set of routes)